spec/features/course_page_spec.rb

```diff
@@ -472,8 +475,11 @@ def js_visit(path, count=3)
 
   describe 'activity view' do
     it 'displays a list of edits' do
-      js_visit "/courses/#{slug}/activity"
-      expect(page).to have_content 'Article 1'
+      Capybara.using_wait_time 10 do
+        js_visit "/courses/#{slug}/activity"
+        expect(page).to have_content('Bareilly College').twice
+        expect(page).to have_content('Hartmann College', count: 3)
```

Comment:
Is this going to break if/when that user has more recent edits to different pages?

Reply:
Just put an expectation of at least X revisions appearing, without looking for any specific ones. That way it will verify that it pulls in revisions, but it will always work regardless of what new revisions happen.